tfjs-converter/src/operations/executors/convolution_executor_test.ts

```diff
@@ -202,14 +201,46 @@ describe('convolution', () => {
         node.op = 'MaxPool3D';
         node.attrParams['strides'] = createNumericArrayAttr([1, 2, 2, 2, 1]);
         node.attrParams['pad'] = createStrAttr('same');
-        node.attrParams['kernelSize'] =
-            createNumericArrayAttr([1, 2, 2, 2, 1]);
+        node.attrParams['kernelSize'] = createNumericArrayAttr([1, 2, 2, 2, 1]);
 
         executeOp(node, {input}, context);
 
         expect(tfc.maxPool3d)
             .toHaveBeenCalledWith(input[0], [2, 2, 2], [2, 2, 2], 'same');
       });
     });
+
+    describe('_FusedConv2d', () => {
+      it('should call tfc.fused.conv2d', () => {
+        spyOn(tfc.fused, 'conv2d');
+        node.op = '_FusedConv2D';
+        node.inputParams['filter'] = createTensorAttr(1);
+        node.inputParams['args'] = createTensorsAttr(2, 0);
+        node.attrParams['fusedOps'] = createStrArrayAttr(['biasadd', 'relu']);
+        node.attrParams['strides'] = createNumericArrayAttr([1, 2, 2, 1]);
+        node.attrParams['pad'] = createStrAttr('same');
+        node.attrParams['dataFormat'] = createStrAttr('NHWC');
+        node.attrParams['dilations'] = createNumericArrayAttr([1, 2, 2, 1]);
+        node.attrParams['numArgs'] = createNumberAttr(1);
+        const input1 = [tfc.scalar(1.0)];
+        const input2 = [tfc.scalar(2.0)];
+        const input3 = [tfc.scalar(3.0)];
+
+        node.inputNames = ['input1', 'input2', 'input3'];
+        executeOp(node, {input1, input2, input3}, context);
+
+        expect(tfc.fused.conv2d).toHaveBeenCalledWith({
+          x: input1[0],
+          filter: input2[0],
+          strides: [2, 2],
+          pad: 'same',
+          dataFormat: 'NHWC',
+          dilations: [2, 2],
+          bias: input3[0],
+          activation: 'relu',
+          preluActivationWeights: undefined
+        });
+      });
+    });
   });
```

Comment:
Should we also add a test for correctness of fused conv2d output?

Reply:
yes make sense, will add.